libpod/boltdb_state.go

```diff
@@ -455,7 +456,7 @@ func (s *BoltState) UpdateContainer(ctr *Container) error {
 		} else {
 			// Tear down the existing namespace
 			if err := s.runtime.teardownNetNS(ctr); err != nil {
-				return err
+				logrus.Warnf(err.Error())
```

Comment:
Should we add a little explanatory text to these new warnings?  I'm not coming up with a concise description though.  Perhaps "Network was likely cleared via other cleanup actions:"  or some such?

Reply:
I looked at their library and they don't have any constants for errors.  Or an API to see if the network was already closed.